src/FNA3D_Driver_Vulkan.c

```diff
@@ -11371,6 +11274,7 @@ static void VULKAN_SetIndexBufferData(
 	int32_t dataLength,
 	FNA3D_SetDataOptions options
 ) {
+	/* FIXME: use staging buffer for elementSizeInBytes < vertexStride */
```

Comment:
Looks like it was added to the wrong one...

Reply:
Hahahaha whoops, one sec